espanol/ritual/ritual.md

```diff
@@ -0,0 +1 @@
+Empezó con el ritual de bienvenida para el ejército, el cual incluía los cánticos milenarios de los ganadores de las mil batallas del gran ejército de la República.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.